modules/core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/AsyncStatus.java

```diff
@@ -38,4 +39,34 @@
      * Returns true if command is completed.
      */
     boolean isCompleted();
+
+    /**
+     * Gets the error message if any.
+     *
+     * @return the error message if any, {@code null} otherwise
+     * @since 11.5
+     */
+    default String getError() {
```

Comment:
I would prefer a `getErrorMessage` to have the same method name as `BulkStatus` and avoid a bridge method.